playbooks/select_kernel_ddp_package.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Select Kernel DDP Package
+  hosts: all
+  strategy: linear
+  become: true
+  vars:
+    edpm_ddp_package: 'ddp'
```

Comment:
why pass the `edpm_ddp_package` variable with its default value?

Reply:
@rebtoor just provided variable initialization with default value to use this playbook.
I Agree, need to remove this section to use playbook without changing. will update the code.